README.md

```diff
@@ -417,6 +566,56 @@ use TLS encryption.
     - microsoft.sql.server
 ```
 
+### Setting Up SQL Server and Configuring for High Availability
+
+This example shows how to use the role to set up SQL Server and configure it for high availability.
+
+You must set the `mssql_ha_replica_type` variable for each host that you want to configure, for example in the inventory file.
+
+Example inventory file with `mssql_ha_replica_type` set for each host:
+
+```yaml
+all:
+  hosts:
+    host1:
+      mssql_ha_replica_type: primary
+    host2:
+      mssql_ha_replica_type: synchronous
+    host3:
+      mssql_ha_replica_type: witness
+```
+
+When the `mssql_ha_replica_type` variable is set for all hosts, you can execute a playbook.
+
+Example playbook:
+
+```yaml
+- hosts: all
+  vars:
+    mssql_accept_microsoft_odbc_driver_17_for_sql_server_eula: true
+    mssql_accept_microsoft_cli_utilities_for_sql_server_eula: true
+    mssql_accept_microsoft_sql_server_standard_eula: true
+    mssql_password: "p@55w0rD"
+    mssql_edition: Evaluation
+    mssql_ha_configure: true
+    mssql_ha_firewall_configure: true
+    mssql_ha_listener_port: 5022
```

Comment:
Another minor question... If you plan to use the default values, you don't have to declare the variables `mssql_ha_firewall_configure` and `mssql_ha_listener_port`, right? Of course, it's ok to set them, though... This question is applied to tests_configure_ha_cluster.yml, as well. Maybe, it's a good practice type of issue...? :) Setting all the available variables vs. setting as little variables as possible.